src/KIM/pair_kim.cpp

```diff
@@ -66,9 +66,13 @@
 #include "neigh_request.h"
 #include "neighbor.h"
 #include "update.h"
+#include "utils.h"
```

Comment:
here, too.

Reply:
@akohlmey I updated the include statements